Oxide.Rust/RustCommands.cs

```diff
@@ -663,5 +663,18 @@ private void VersionCommand(IPlayer player, string command, string[] args)
         }
 
         #endregion Version Command
+
+        #region Save Command
+
+        [HookMethod("SaveCommand")]
+        private void SaveCommand(IPlayer player, string command, string[] args)
+        {
+            if (!PermissionsLoaded(player)) return;
+
+            Interface.Oxide.OnSave();
+            player.Reply(lang.GetMessage("OxideSaved", this, player.Id));
```

Comment:
Should not contain "Oxide" in the key, would be better as "SavingData" and "Saving Oxide data..." and likely be good to add a "SavedData" key as well for post save.